CONTRIBUTING.md

```diff
@@ -1,62 +1,172 @@
+
+This project follows the [Conventional Commits][] specification.
+
+Commits should be signed. You can read more about [Commit Signing][] here.
+
+### Commit Message guidelines
+
+- Commit messages must include a "type"
+  > build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test
+- Commit messages **must** start with a capital letter
+- Commit messages **must not** end with a period `.`
+
+## Pull Request Policy
+
+### Before merging
+
+- Pull Requests must be open for at least 48 hours.
+  - Pull requests might be immediately merged if they contain critical bug fixes, short erratas (e.g. typos from previous PRs) or any critical change that is considered a "showstopper" for the operation of the website.
+    - These kind of PRs should only be done by existing collaborators that have write-access and/or signed-off by administrators/maintainers.
+- There must be no objections after a 48 hour period
+- Tests must be included in Pull Requests for new features or bug fixes. If any test(s) are failing, you are responsible for fixing them.
+
+Each contribution is accepted only if there is no objection to it by a collaborator. During the review, collaborators may request that a specific contributor who is an expert in a particular area give an "LGTM" before the PR can be merged.
+
+In the case that an objection is raised in a pull request by another collaborator, all collaborators involved should try to arrive at a consensus by addressing the concerns through discussion, compromise, or withdrawal of the proposed change(s).
+
+### When merging
+
+- [`squash`][] pull requests made up of multiple commits
+
+## Vocabulary
 
-No pull request can be merged without being reviewed.
+- A **Contributor** is any individual who creates an issue/PR, comments on an issue/PR
+  or contributes in some other way.
+- A **Collaborator** is a contributor with write access to the repository. See [here](#becoming-a-collaborator) on how to become a collaborator.
 
-For non-trivial contributions, pull requests should sit for at least 36 hours to ensure that
-contributors in other timezones have time to review. Consideration should also be given to
-weekends and other holiday periods to ensure active collaborators all have reasonable time to
-become involved in the discussion and review process if they wish.
+## Becoming a collaborator
 
-The default for each contribution is that it is accepted once no collaborator has an objection.
-During review collaborators may also request that a specific contributor who is most versed in a
-particular area gives a "LGTM" before the PR can be merged. There is no additional "sign off"
-process for contributions to land. Once all issues brought by collaborators are addressed it can
-be landed by any collaborator.
+- Collaborators must be actively contributing to the project
+- A Pull Request must be opened on the @nodejs/nodejs.org README file adding the new collaborator to the list (note the order of names)
```

Comment:
AFAIS collaborator section is not included in current README. See: https://github.com/nodejs/nodejs.org/blob/main/README.md

Reply:
Yup, that's correct. It will in the future :)